pkg/defaults/defaults.go

```diff
@@ -116,8 +116,12 @@ const (
 	// DefaultCgroupRoot is the default path where cilium cgroup2 should be mounted
 	DefaultCgroupRoot = "/run/cilium/cgroupv2"
 
-	// SockopsEnable controsl whether sockmap should be used
-	SockopsEnable = false
+	// DefaultMapRootFallback is the path which is used when /sys/fs/bpf has
+	// a mount, but with the other filesystem than BPFFS.
+	DefaultMapRootFallback = "/run/cilium/bpffs"
+
+	// DefaultMapPrefix is the default prefix for all BPF maps.
+	DefaultMapPrefix = "tc/globals"
```

Comment:
Looks like something went wrong with the rebase. These shouldn't be added again. They were dropped in commit b2865c5c0d1e557929319cdb74bd97cf12289559.